doc/en/user/source/community/vectortiles/tutorial.rst

```diff
@@ -0,0 +1,237 @@
+.. _vectortiles.tutorial:
+
+Vector tiles tutorial
+=====================
+
+This tutorial will show how to use the GeoServer vector tiles output.
+
+Why use vector tiles?
+---------------------
+
+The advantages of vector tiles are;
+
+* **Rendering is done by the client** (for example, OpenLayers), not by the server. This allows different maps/applications to style a map differently without having to reconfigure GeoServer.
+
+* **The size of a vector tile is usually smaller** than an image tile, resulting in faster data transfer and lower bandwidth usage.
+
+* GeoWebCache, embedded with GeoServer efficiently stores the vector tile data. Since styling is done by the client, not the server, **GeoWebCache only needs to store one tile for all different styles**.
+
+* Because the vector data is available on the client, very **high-resolution maps can be drawn without corresponding increases in bandwidth**.
+
+* **The client has native access to the actual feature information** (attributes and geometry), allowing for very sophisticated rendering.
+
+On the other hand, the main disadvantage of vector tiles is that the geographic data may need to be pre-processed to allow the client to do the drawings it requires (similar to preprocessing data for image maps). With this in mind, **vector tiles should only be used for rendering**.
+
+Vector tile formats
+-------------------
+
+GeoServer can also produce vector tiles in three formats: GeoJSON, TopoJSON, and MapBox Vector (MVT). These are also supported by OpenLayers 3 and other clients.
+
+* GeoJSON is the most human readable and the recommended format for people new to vector tiles.
+* MVT is the preferred format for production.
+
+.. list-table::
+   :header-rows: 1
+   :class: non-responsive
+
+   * - Format
+     - MIME
+     - Description
+   * - `GeoJSON <http://geojson.org/>`_
+     - ``application/json;type=geojson``
+     - Human readable, format (not tiles) widely supported
```

Comment:
"format (not tiles)" --> what does this mean? If we ask for a tile don't we get back only what's in the tile, even if the format is GeoJSON?